src/script/ceph-release-notes

```diff
@@ -62,6 +63,24 @@ def make_release_notes(gh, repo, ref, plaintext):
                 print ("ERROR: http://github.com/ceph/ceph/pull/" + str(number) + " has no associated issue")
                 continue    
 
+            if original_issue:
+                r = requests.get(tracker_uri.format(issue),params={"include":"relations"})
+
+                # Verify that we are indeed looking at a backports issue
+                if r.json()["issue"]["tracker"]["name"] != "Backport":
+                    continue
+
+                # Now we can get an original issue only if the
+                # relations field is populated, otherwise we fallback
+                # to the tracker number
+                try:
+                    relations = r.json()["issue"]["relations"]
+                except KeyError:
+                    continue
```

Comment:
if a Backport issue has no relation it will be skipped, i.e. the issue2prs.setdefault below won't be set